### DISCUSSION TITLE
How to emulate Android Apps on Desktop

### DISCUSSION THREAD

**Discussion by mahmoodtejpar** [community/community#37160] [Other Feature Feedback, Questions, & Ideas]
Hello Team
As some apps are only available on Mobile for example www.hopper.com How can I emulate my Pixel Android App on Desktop PC Windows?
Any help will be greatly appreciated. If you can email me please and thank you [removed by moderator]

**Comment by ossanna16** [comment]
Hi @mahmoodtejpar,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Android Apps. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by mahmoodtejpar** [reply]
Thank you for taking the time to help and guide me! Any chance where would I find Topics under Android? Sorry I am new to the forum and or the GitHub Community questions. Is there a Youtube Tutorial on this. Again thank you for your time God Bless 
best Regards
Mahmood

**Reply by ossanna16** [reply]
Hi @mahmoodtejpar! No worries at all 🙂 I don't think the GitHub Community is the best place for you to get help with Android related questions. I recommend doing a Google search on any forums specifically dedicated to Android topics.

**Reply by mahmoodtejpar** [reply]
Anna now that you mention GitHub for Project Development, I do have something I would like to know about how, and where I could post this request on GitHub. This would be Joint Venture and or collaboration? I would like to Create a Community Help page similar to http://www.shepherdsguide.ca/
[removed by moderator]

**Reply by ossanna16** [reply]
@mahmoodtejpar Please post your idea in the General category.

**Comment by tawfik-s** [comment]
you can use an android emulator on your desktop like
bluestacks
or you can use WSA  (windows subsystem for Android) if you have windows 11

**Reply by tawfik-s** [reply]
verify as an answer if it helps you.

**Reply by Wmfbtk** [reply]
I would try out bluestack my friend..

**Reply by mahmoodtejpar** [reply]
Thank you Will check it out.
